hoot-core-test/src/test/cpp/hoot/core/conflate/point/PlacesPoiMatchTest.cpp

```diff
@@ -162,7 +162,7 @@ class PlacesPoiMatchTest : public CppUnit::TestFixture
         shared_ptr<const Element> e1 = map->getElement(it->first);
         shared_ptr<const Element> e2 = map->getElement(it->second);
         //LOG_INFO(e1->getTags()["note"] << " <=> " << e2->getTags()["note"]);
-        CPPUNIT_ASSERT_EQUAL(e1->getTags()["note"], e2->getTags()["note"]);
+        CPPUNIT_ASSERT_EQUAL(e1->getTags()["note"].toStdString(), e2->getTags()["note"].toStdString());
```

Comment:
You can use HOOT_STR_EQUALS to avoid the .toStdString() call. Handy for other comparisons too.